content/doc/book/installing/_jenkins-command-parameters.adoc

```diff
@@ -139,27 +139,51 @@ the keystore to PEM] and use following command line options:
 
 === Using HTTP/2
 
-With Java 8 will not use HTTP/2 unless alpn boot jar is included in the bootclasspath.
-The alpn boot jar depends on your jvm version.
-Refer to the link:https://www.eclipse.org/jetty/documentation/current/alpn-chapter.html#alpn-versions[Jetty documentation] to select the required version.
+The link:https://tools.ietf.org/html/rfc7540[HTTP/2 protocol] allows web servers to reduce latency over encrypted connections by pipelining requests, multiplexing requests, and allowing servers to push in some cases before receiving a client request for the data.
+The Jetty server used by Jenkins supports HTTP/2 with the addition of the Application-Layer Protocol Negotiation (ALPN) TLS extension.
+The ALPN TLS extension is connected to the specific Jetty version and has specific requirements depending on the Java version.
 
-You can download the required version from the link:https://repo.maven.apache.org/maven2/org/mortbay/jetty/alpn/alpn-boot/[Apache maven repository].
+==== Java 8u252 and Java 11
 
-Then you have to include it on jvm start:
+Java 8 update 252 (and later) and all versions of Java 11 can run the ALPN TLS extension by installing the Jetty ALPN java server jar and passing it as a java command line argument.
+Steps to install the extension are:
+
+* Identify the Jetty version included in your Jenkins server by reviewing the link:/changelog/[Jenkins changelog] for your release
+* Locate the Java version on the "System Information" page of "Manage Jenkins" to confirm it is Java 11 or 8u252 (or later)
+* Download the link:https://mvnrepository.com/artifact/org.eclipse.jetty/jetty-alpn-java-server[jetty-alpn-java-server] with the version number matching the Jetty version bundled with your Jenkins version
+* Place the jetty-alpn-java-server.jar file in a directory accessible to the JVM
+* Add `--extraLibFolder=/path/to/extra/lib/folder` to the Java command line arguments that start Jenkins
+
+[source,bash]
+----
+java --extraLibFolder=/opt/java/jetty-alpn-java-server-9.4.27.v20200227.jar \
+    -jar target/jenkins.war \
+    --http2Port=9090
+----
+
+==== Java 8u242 and earlier
+
+Java 8 update 242 (and earlier) can run the ALPN TLS extension by installing the Jetty ALPN java server jar into the Java boot classpath.
+Steps to install the extension are:
+
+* Identify the Jetty version included in your Jenkins server by reviewing the link:/changelog/[Jenkins changelog] for your release
```

Comment:
we should keep this link in the doc https://www.eclipse.org/jetty/documentation/current/alpn-chapter.html#alpn-versions as it may ease users life? 
now jetty version is displayed at start `org.eclipse.jetty.server.Server#doStart: jetty-9.4.22.v20191022` maybe we can add a note here (how to find the Jetty version)

Reply:
Is there a reason this isn’t packaged with winstone already? It’s not terribly useable having to keep it synced manually (which is why we aren’t using it)

Reply:
I suspect that it is not packaged with winstone because winstone delivers a single jar file and can't readily unpack another jar to be referenced from the Java command line that starts Jenkins.

I think it would be feasible to include it in the Docker images and to configure the Docker images to optionally use it.  I'll  submit an enhancement request suggesting it.  It seems like that would be a good way to test SSL encryption within the JDK.

Reply:
Changes suggested by @olamy have been included in the latest text.  Would love to have approval to merge this.  Approaching the longest documentation pull request I've ever created to remove a single sentence.